### ISSUE TITLE
Not access to folders

### ISSUE THREAD

**Issue by RastaDill** [exfatprogs/exfatprogs#198]
Hello. I have:
* Fedora 35
* exitprocess v1.1.3
* external USB Hard Drive

In my case USB Hard Drive shared folders via my router. When i mount hard drive to my PC a have not access to my folder:
* If open via file manager - mount point show is empty
* if i  do command ```ls``` - no it is not possible to access the "Game": there is no such file or directory
Games is my exists folder which success share via router.
superuser not helped.

I tryed mount use command:
```sudo mount -t exfat /dev/sda1 /media/ex/```
or
```sudo mount -o rw,users,uid=1000,gid=1000,dmask=000,fmask=000 /dev/sda1 /media/ex/```
but does not sense.
In Windows the hard drive work corrected.
fsck.exfat works out correct - no errors.

How fix this problem?

**Comment by namjaejeon** [comment]
Hm.. I just guessing this problem is name hash.

Can you check the below patch ? Let me know if your issue is fixed when using skip_stream_check mount option.
https://lore.kernel.org/lkml/CAKYAXd99NAbQP6m93P3bcjvWTN-T8Qy59DHJyfyTHqdH-7aWBQ@mail.gmail.com/T/

**Comment by namjaejeon** [comment]
Please do the following step.

git clone --branch=test https://github.com/namjaejeon/linux-exfat-oot

```
cd linux-exfat-oot
make
insmod exfat.ko

sudo mount -t exfat /dev/sda1 /media/ex/
```

**Comment by RastaDill** [comment]
@namjaejeon Something is not working to apply the patch.
log out make:
```
make -C /lib/modules/5.18.10-100.fc35.x86_64/build M=/home/rastaukropchik/Code/GitHub/linux-exfat-oot modules
make[1]: вход в каталог «/usr/src/kernels/5.18.10-100.fc35.x86_64»
  CC [M]  /home/rastaukropchik/Code/GitHub/linux-exfat-oot/inode.o
  CC [M]  /home/rastaukropchik/Code/GitHub/linux-exfat-oot/namei.o
  CC [M]  /home/rastaukropchik/Code/GitHub/linux-exfat-oot/dir.o
  CC [M]  /home/rastaukropchik/Code/GitHub/linux-exfat-oot/super.o
  CC [M]  /home/rastaukropchik/Code/GitHub/linux-exfat-oot/fatent.o
  CC [M]  /home/rastaukropchik/Code/GitHub/linux-exfat-oot/cache.o
  CC [M]  /home/rastaukropchik/Code/GitHub/linux-exfat-oot/nls.o
  CC [M]  /home/rastaukropchik/Code/GitHub/linux-exfat-oot/misc.o
  CC [M]  /home/rastaukropchik/Code/GitHub/linux-exfat-oot/file.o
  CC [M]  /home/rastaukropchik/Code/GitHub/linux-exfat-oot/balloc.o
  LD [M]  /home/rastaukropchik/Code/GitHub/linux-exfat-oot/exfat.o
  MODPOST /home/rastaukropchik/Code/GitHub/linux-exfat-oot/Module.symvers
  CC [M]  /home/rastaukropchik/Code/GitHub/linux-exfat-oot/exfat.mod.o
  LD [M]  /home/rastaukropchik/Code/GitHub/linux-exfat-oot/exfat.ko
  BTF [M] /home/rastaukropchik/Code/GitHub/linux-exfat-oot/exfat.ko
Skipping BTF generation for /home/rastaukropchik/Code/GitHub/linux-exfat-oot/exfat.ko due to unavailability of vmlinux
make[1]: выход из каталога «/usr/src/kernels/5.18.10-100.fc35.x86_64»
```
Next spet do insmod via sudo
```
insmod: ERROR: could not insert module exfat.ko: Key was rejected by service
```

I tryed on my laptop, not virtual machine.

**Comment by namjaejeon** [comment]
@RastaDill I don't know how I can help you. You need to find why you can not load exfat.ko in your system.

**Comment by namjaejeon** [comment]
@RastaDill Can you run fsck.exfat to your exfat device ?

You need to download exfat-next branch, which is not release yet.

git clone --branch=exfat-next https://github.com/exfatprogs/exfatprogs

**Comment by RastaDill** [comment]
@namjaejeon i clone, build and install exfatprogs:
```
→ sudo fsck.exfat /dev/sda1 
exfatprogs version : 1.1.3
ERROR: /: the name hash of a file is wrong at 0xb80180. Fix (y/N)? n
ERROR: /: the name hash of a file is wrong at 0xb80240. Fix (y/N)? n
/dev/sda1: corrupted. directories 4, files 3
/dev/sda1: files corrupted 2, files fixed 0
```
After add key -y:
```
/dev/sda1: clean. directories 197, files 1623
/dev/sda1: files corrupted 0, files fixed 429
```

fsck fixed all file on folders which was not accessible. Now after mount all corrected!
Updated bug on Fedora bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2113964

**Comment by namjaejeon** [comment]
@RastaDill Thanks for your check! We will release public next version soon.

**Comment by scaronni** [comment]
> @RastaDill Thanks for your check! We will release public next version soon.

Hello! Any chance for an ETA for the new release? Thanks.

**Comment by namjaejeon** [comment]
There have been additional fixes recently. We are planning to release 1.2.0 version next week.

**Comment by namjaejeon** [comment]
@RastaDill I have released 1.2.0 version. Please close this ISSUE if you don't have other issue.
